minio/minio
# Suppress error log for force-deleting object in locked bucket

## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description
Suppress error message for force-deleting an object in a lock-enabled bucket.

## Motivation and Context
MinIO is already returning the correct response, therefore, we don't need to return this error message.

## How to test this PR?
Verify that the error message is not being seen when trying to force-delete an object in a bucket with locking enabled.

## Types of changes

## Checklist:
